runtime/rasdump/javadump.cpp

```diff
@@ -2526,7 +2545,7 @@ JavaCoreDumpWriter::writeHookSection(void)
 	/* Write the section header */
 	_OutputStream.writeCharacters("0SECTION       HOOK subcomponent dump routine\n");
 	_OutputStream.writeCharacters("NULL           ==============================\n");
-
+	_OutputStream.writeCharacters("1NOTE		  This data is reset every time a javacore is taken\n");
```

Comment:
Please don't embed tabs within string literals: use spaces for consistent alignment.

Reply:
My mistake. The string now contains only spaces